.github/workflows/regression.yml

```diff
@@ -773,16 +777,17 @@ jobs:
           status: 'success'
 
       - name: (PR) Indicate why skipped
-        if: github.event_name == 'pull_request'
+        if: github.event_name == 'pull_request' || env.SKIP_REGRESSION == 'true'
         run: |
-           echo "### Workflow skipped" >> $GITHUB_STEP_SUMMARY
-           echo "This workflow doesn't run on PR's automatically." >> $GITHUB_STEP_SUMMARY
-           echo "To trigger a run, leave a comment with \`/ci-run-regression\`" >> $GITHUB_STEP_SUMMARY
+          echo "### Workflow skipped" >> $GITHUB_STEP_SUMMARY
+          echo "This workflow doesn't run on PRs automatically." >> $GITHUB_STEP_SUMMARY
+          echo "This workflow doesn't run on PRs created by Dependabot." >> $GITHUB_STEP_SUMMARY
+          echo "To trigger a run, leave a review comment with \`/ci-run-regression\`" >> $GITHUB_STEP_SUMMARY
 
       - name: exit
         run: |
           echo "failed=${{ env.FAILED }}"
-          if [[ "$FAILED" == "true" ]] ; then
```

Comment:
Was this change necessary? The previous line could probably be deleted if so.

Reply:
No, not necessary. In fact I will revert it since it's not relevant to this PR.

Reply:
[revert unrelated change](https://github.com/vectordotdev/vector/pull/21507/commits/f4064364f4121bf95d6be779b2896c4dff39b0a2)